openstef/model/serializer.py

```diff
@@ -81,11 +86,13 @@ def save_model(
             self._log_figure_with_mlflow(report)
         self.logger.debug(f"MLflow path after saving= {self.mlflow_folder}")
 
-        # Also store report files in easy-to-find location, which are updated on each new model
-        # Easy for web visualisation, e.g. through grafana
-        location = os.path.join(self.web_volume, f'{pj["id"]}')
-        Reporter.write_report_to_disk(report, location=location)
-        self.logger.info(f"Stored report to disk: {location}")
+        if save_report_to_disk:
```

Comment:
Good idea to make this an option.

Reply:
Yes that makes it possible to not use a persistent volume